modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/IgniteCacheDatabaseSharedManager.java

```diff
@@ -18,11 +18,15 @@
 package org.apache.ignite.internal.processors.cache.persistence;
 
 import java.io.File;
+import java.text.DecimalFormat;
```

Comment:
Do we really need _visual representation_ stuff in `IgniteCacheDatabaseSharedManager`? Can we extract it somewhere?